setup.py

```diff
@@ -182,7 +182,17 @@ def wrapper(*args, **kw):  # type: ignore
 _get_beacon_committee = get_beacon_committee
 get_beacon_committee = cache_this(
     lambda state, slot, index: (state.validators.hash_tree_root(), state.randao_mixes.hash_tree_root(), slot, index),
-    _get_beacon_committee)'''
+    _get_beacon_committee)
+
+_get_matching_target_attestations = get_matching_target_attestations
+get_matching_target_attestations = cache_this(
```

Comment:
As we start to memoize more functions we should update the `cache_this` helper to cache up to some maximum N items. Maybe with an LRU cache. Something for after v0.11.0 I think, this works well enough for now.